package.json

```diff
@@ -1,37 +1,87 @@
 {
   "name": "testcafe-reporter-isaac",
-  "version": "0.1.0",
+  "version": "0.0.0-develop",
+  "author": "ISAAC E-commerce Solutions BV",
+  "license": "MIT",
   "description": "An Allure reporter plugin for TestCafé. Used to generate test reports for e2e-tests.",
-  "main": "index.js",
+  "keywords": [
+    "testcafé",
+    "allure",
+    "integration"
+  ],
+  "main": "index.ts",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/isaaceindhoven/testcafe-reporter-isaac.git"
   },
-  "author": "ISAAC E-commerce Solutions BV",
-  "license": "MIT",
   "bugs": {
     "url": "https://github.com/isaaceindhoven/testcafe-reporter-isaac/issues"
   },
   "homepage": "https://github.com/isaaceindhoven/testcafe-reporter-isaac#readme",
   "scripts": {
+    "build": "tsc",
```

Comment:
When running the build script, it seems the tests are also compiled. This seems unnecessary as jest is using ts-jest to directly use the TypeScript test files.

This behavior is logical. The tsconfig.json says that test-files should be included.
https://github.com/isaaceindhoven/testcafe-reporter-isaac/blob/bda9bb29dd355615816b9ab50243563ff799111b/tsconfig.json#L12-L15

### How can we ignore these files?
#### Don't include or even exclude them in the tsconfig.json. 
This seems the simplest option, but it gives us the unwanted side effect that also ESLint will then ignore these files. `typescript-eslint` uses the tsconfig to parse and lint TypeScript files.

https://github.com/isaaceindhoven/testcafe-reporter-isaac/blob/bda9bb29dd355615816b9ab50243563ff799111b/.eslintrc.js#L3-L5

#### Use a separate tsconfig.
You can assign a separate config for the TypeScript build. One with the base configuration with inclusion* of your test-files for `ts-jest` and `eslint` and one where you or exclude* your test files.

* or vice-versa

Eg.
https://github.com/vanhoofmaarten/vue-transition-a11y/blob/develop/tsconfig.base.json
https://github.com/vanhoofmaarten/vue-transition-a11y/blob/develop/tsconfig.prod.json

Reply:
Created separate tsconfig files for both testing and building, namely `tsconfig.test.json` and 'tsconfig.prod.json'. Both inherit from the same base, 'tsconfig.base.json'. This was done this way because both the test and prod versions include and exclude different files. Instead of one extending the other, both extending from a base configuration appears cleaner.